digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1786,7 +1783,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     * if any such project is waiting.
     * @throws SQLException if there is an error with preserving the reserved project to the db.
     * @throws Exception if an unexpected exception occurred. */
-  def atomicallyPreserveSingleProjectInUpdateQueue(): Unit = {
+  def preserveSingleProjectToBeTakenToRoadNetwork(): Unit = {
```

Comment:
Onko tää trakoituksella preserve vai pitäisikö olla reserve?

Reply:
Tarkoituksella.  ("/** Preserves the information of ---" )
Mutta hämääkö? Oisko ihan vaan save-- parempi?